packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -37,9 +38,14 @@ class DatadogTracer extends Tracer {
     this._logInjection = config.logInjection
     this._analytics = config.analytics
     this._prioritySampler = new PrioritySampler(config.env)
-    this._writer = new Writer(this._prioritySampler, config.url)
-    this._recorder = new Recorder(this._writer, config.flushInterval)
-    this._recorder.init()
+
+    if (config.experimental !== undefined && config.experimental.useLogWriter) {
```

Comment:
This should be `useLogExporter`. I wonder if the exporter could be configurable instead, something like `exporter: LOG_EXPORTER`.

Reply:
I've updated this, I also export those constants in the module, and in the d.ts file